test/e2e/framework/azureidentity/azureidentity_helpers.go

```diff
@@ -9,6 +9,8 @@ import (
 	"io/ioutil"
 	"strings"
 
+	api "k8s.io/api/core/v1"
+
```

Comment:
nit: import order should be

```
import(
  <standard libs>

  <internal libs>

  <external libs>
)
```

Reply:
yeah, Goland IDE does it in a different way. Let me fix it.